src/main/java/ch/njol/skript/tests/platform/Environment.java

```diff
@@ -220,14 +219,15 @@ public void initialize(Path dataRoot, Path runnerRoot, boolean remake) throws IO
 		}
 	}
 
-	public TestResults runTests(Path runnerRoot, Path testsRoot, boolean devMode, String... jvmArgs) throws IOException, InterruptedException {
+	public TestResults runTests(Path runnerRoot, Path testsRoot, boolean devMode, boolean genDocs, String... jvmArgs) throws IOException, InterruptedException {
 		Path env = runnerRoot.resolve(name);
 		List<String> args = new ArrayList<>();
 		args.add(System.getProperty("java.home") + File.separator + "bin" + File.separator + "java");
 		args.add("-ea");
 		args.add("-Dskript.testing.enabled=true");
 		args.add("-Dskript.testing.dir=" + testsRoot);
 		args.add("-Dskript.testing.devMode=" + devMode);
+		args.add("-Dskript.testing.genDocs=" + genDocs);
 		args.add("-Dskript.testing.results=test_results.json");
```

Comment:
We need a place to store the new docs as well

Reply:
It goes into test_runner/ENV/plugins/Skript

Reply:
Okay cool then!